src/plugins/intel_cpu/src/emitters/snippets/x64/cpu_generator.cpp

```diff
@@ -157,6 +178,8 @@ intel_cpu::CPUTargetMachine::CPUTargetMachine(dnnl::impl::cpu::x64::cpu_isa_t ho
     jitters[snippets::op::PerfCountEnd::get_type_info_static()] = CREATE_CPU_EMITTER(ov::intel_cpu::jit_perf_count_chrono_end_emitter);
     jitters[ov::intel_cpu::PerfCountRdtscBegin::get_type_info_static()] = CREATE_CPU_EMITTER(ov::intel_cpu::jit_perf_count_rdtsc_start_emitter);
     jitters[ov::intel_cpu::PerfCountRdtscEnd::get_type_info_static()] = CREATE_CPU_EMITTER(ov::intel_cpu::jit_perf_count_rdtsc_end_emitter);
+    DebugCapsConfig debugCaps;
```

Comment:
Applied in general. Created independent snippets debug config. As jitters responsible for emitters creation is in target machine, so leave snippets debug config field there. Subgraph node can still access it from `get_generator()->get_target_machine()`, otherwise we need redundant in generator and target. Do you think it make sense? Thank you!